### PR TITLE
Added seed packs to loot tracker

### LINKED ISSUE #10604: seed packs to the loot tracker
would be a nice addition- i know miscellania got added recently.

### PR DESCRIPTION
**Pull request by cscullen**
Adds seeds taken from seed packs into loot tracker
Closes #10604
image

KC is tracked as per 'take' action so a single seed pack could have 6-10 entries each. If multiple seeds are taken in the same tick then the kc will not be correct, but loot will be properly tracked still. Since I would assume most people only care about profits I don't think these issues are a problem.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 1]
```diff
@@ -631,6 +636,14 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 		{
 			lastPickpocketTarget = Text.removeTags(event.getMenuTarget());
 		}
+
+		if (event.getMenuOption().equals(("Take")) && event.getId() == SEEDPACK_ID)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java:640]:
```suggestion
		if (event.getMenuOption().equals("Take") && event.getId() == SEEDPACK_ID)
```

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 2]
```diff
@@ -676,7 +689,7 @@ private void takeInventorySnapshot()
 		{
 			inventorySnapshot = HashMultiset.create();
 			Arrays.stream(itemContainer.getItems())
-					.forEach(item -> inventorySnapshot.add(item.getId(), item.getQuantity()));
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java:679]: try to avoid unnecessary reformatting of unrelated code

File: runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java [hunk 3]
```diff
@@ -126,6 +126,10 @@
 	private static final String HERBIBOAR_EVENT = "Herbiboar";
 	private static final Pattern HERBIBOAR_HERB_SACK_PATTERN = Pattern.compile(".+(Grimy .+?) herb.+");
 
+	//Seed Pack loot handling
+	private static final String SEEDPACK_EVENT = "Seed pack";
+	private static final int SEEDPACK_ID = 22993;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java:131]: Instead of defining this constant, use `ItemID.SEED_PACK`.
